sway/config.c

```diff
@@ -389,7 +389,7 @@ bool load_main_config(const char *file, bool is_active, bool validating) {
 	config_defaults(config);
 	config->validating = validating;
 	if (is_active) {
-		wlr_log(WLR_DEBUG, "Performing configuration file reload");
+		wlr_log(WLR_DEBUG, "Performing configuration file reload or validation");
```

Comment:
Can you use `validating` to display a more precise debug message?

Reply:
Done.